js/src/main/scala/org/widok/bindings/Bootstrap/package.scala

```diff
@@ -141,6 +148,8 @@ package object Bootstrap {
     def element = DOM.createElement("button", contents)
   }
 
+  def SubmitButton(contents: View*) = Button(contents: _*).attribute("type", "submit")
```

Comment:
We already have ``Button`` in ``Bootstrap``, which inherits from ``ButtonBase`` and thus has the classes ``btn btn-default``. We could probably define ``ButtonBase`` in ``HTML``, so that ``type`` is available on HTML and Bootstrap buttons.